data/hamidayb.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "Sara Jaoude",
+  "type": "personal",
+  "bio": "Lawyer turned **Digital Nomad**. Travel enthusiast Yoga & Peloton aficionado"
+  "avatar": "https://github.com/hamidayb.png",
```

Comment:
@hamidayb You can remove the avatar.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You.